recipes-core/packagegroups/packagefeed-ni-extra.bb

```diff
@@ -65,6 +65,7 @@ RDEPENDS_${PN} = "\
 # meta-nilrt
 RDEPENDS_${PN} += "\
 	packagegroup-ni-debug-kernel \
+	packagegroup-ni-desirable \
```

Comment:
The build steps you've described above (--continue...) don't seem straightforward, and would need insider knowledge. For that reason, I don't like how this feed is structured. I don't have any immediate alternate suggestions either.

Reply:
My intention was that an uninformed builder wouldn't *have* to know that we build one with `--continue` and not the other. That's an implementation detail that we care about, and if the builder wants to do it exactly as we do, they can just run the [`build.extra-feeds.sh`](https://github.com/ni/nilrt/blob/nilrt/master/hardknott/scripts/pipelines/build.extra-feeds.sh) script, and it will do it for you.